src/control/go.mod

```diff
@@ -5,7 +5,6 @@ module github.com/daos-stack/daos/src/control
 // - debian packaging version checks: debian/control
 // Scons uses this file to extract the minimum version.
 go 1.21
-toolchain go1.23.0
```

Comment:
Needs to be reverted. Is there an automated setting in your IDE/environment that keeps removing this?

Reply:
It depends on how intrusive these changes are. Are you proposing skipping parts of CI? Generally the simplest thing is to run at least the default CI stages so we don't have to unnecessarily force land